06-find.md

```diff
@@ -241,11 +241,38 @@ a `tools` directory that contains the programs `format` and `stats`,
 and an empty subdirectory called `old`.
 
 For our first command,
-let's run `find . -type d`.
+let's run `find .`.
+
+~~~ {.bash}
+$ find . -type d
```

Comment:
This looks fine in the context of the changes, -type f is covered afterwards.

If the types are to be removed, there needs to be a bit more extensive changes.